matrix/src/dense.rs

```diff
@@ -300,6 +301,13 @@ impl<'a, T> RowMajorMatrixViewMut<'a, T> {
         self.values.par_chunks_exact_mut(self.width)
     }
 
+    pub fn row_chunks_exact_mut(&mut self, size: usize) -> ChunksExactMut<T>
```

Comment:
There's also a similar `row_chunks_mut` in `RowMajorMatrix`, could we change one or the other so they match?

I.e. the difference seems to be returning slices vs wrapping them in `RowMajorMatrixViewMut`s. The latter might be convenient, but if you find it adds overhead then I think slices are fine.

Reply:
I think both should probably also have `par` in the name, not sure why I hadn't done that with `row_chunks_mut`

Reply:
Refactor to match these two methods.

Reply:
The only difference, is that for `RowMajorMatrixViewMut`, the output is 

```rs
impl IndexedParallelIterator<Item = &mut [T]>
```

instead of

```rs
impl IndexedParallelIterator<Item = RowMajorMatrixViewMut>
```

which eases the code later on, and from a sematics perspective, I think it makes sense to get a `&mut [T]` out from a mutable view matrix. Happy to refactor, if you disagree.